kratos/sources/model_part_io.cpp

```diff
@@ -138,10 +131,11 @@ std::size_t ModelPartIO::ReadNodesNumber()
 
 void ModelPartIO::WriteNodes(NodesContainerType const& rThisNodes)
 {
-    (*mpStream) << "Begin Nodes" << std::endl;
-    for(NodesContainerType::const_iterator it_node = rThisNodes.begin() ; it_node != rThisNodes.end() ; it_node++)
-        (*mpStream) << "\t" << it_node->Id() << "\t" << it_node->X()  << "\t" << it_node->Y() << "\t" << it_node->Z() << std::endl;
-    (*mpStream) << "End Nodes" << std::endl << std::endl;
+    mpStream->precision(16);
```

Comment:
```suggestion
    mpStream->precision(12);
```

I think 12 is enough here, otherwise it will IMO increase the file size to much

I think in other places we use only 10

@vgudzulic is this enough for you?

Reply:
If I change it, I also need to change the test, just mentioning

Reply:
Yes, more than ten is totally sufficient, thanks

Reply:
OK

Reply:
Nope